critiquebrainz/db/review.py

```diff
@@ -522,21 +540,19 @@ def list_reviews(*, inc_drafts=False, inc_hidden=False, entity_id=None, entity_t
                                 language=language, exclude=exclude, sort=sort, limit=limit, offset=offset)
 
 
-def get_popular(limit=None):
+def get_popular_reviews_for_index():
     """Get a list of popular reviews.
 
     Popularity is determined by 'popularity' of a particular review. popularity is a
```

Comment:
not directly related to this change, but this is a weird comment. "popularity is measured by getting the popularity".
We could probably delete this whole first sentence.